tsconfig.json

```diff
@@ -16,12 +16,12 @@
     "baseUrl": "./",
     /* paths - import rewriting/resolving */
     "paths": {
-      "#src/*": [
-        "./src/*"
-      ],
       "#components/*": [
         "./src/components/*"
       ],
+      "#src/*": [
+        "./src/*"
+      ],
```

Comment:
@dbudzins the order is changed so that VS Code prefers the `#components` import over the `#src` import.